src/AutoRest.CSharp/Common/Output/Models/ClientMethod.cs

```diff
@@ -21,5 +21,9 @@ public ClientMethod(string name, RestClientMethod restClientMethod, string? desc
         public string? Description { get; }
         public Diagnostic Diagnostics { get; }
         public string Accessibility { get; }
+
+        public bool IsPut => this.RestClientMethod.IsPut;
```

Comment:
Please remove these properties from the `ClientMethod` type. This type describes the features of the generated method, not its underlying request or requests